sdk/src/loom2d/Events/Touch.ls

```diff
@@ -233,6 +240,9 @@ package loom2d.events
         public function setTapCount(value:int):void { mTapCount = value; }
         
         /** @private */
+        public function setClick(value:Boolean):void { mClicked = value; }
```

Comment:
It is not, but I needed to simulate the Actionscript MouseEvent.CLICKED, which is a Down+Up together which seemed pretty similar to the tapCount, except that it needed the extra Up event tracked, so it made sense to me to add this here.

From what I can see, Starling doesn't support this functionality :/